src/kirby/components/list/list.component.scss

```diff
@@ -3,61 +3,90 @@
 $divider-color: get-color('background-color');
 $background-color-hover: get-color('background-color');
 $shadow: get-elevation(2);
+$item-height: size('xxxl');
 
-:host {
-  ion-list {
-    box-shadow: $shadow;
-    background: transparent;
-    contain: inherit;
-    padding: 0;
+ion-list {
+  box-shadow: $shadow;
+  background: transparent;
+  contain: inherit;
+  padding: 0;
+}
+
+ion-item {
+  min-height: $item-height;
+  background-color: get-color('white');
+  &:not(:last-of-type).has-divider {
+    border-bottom: 1px solid $divider-color;
   }
 
-  ion-item {
-    background-color: get-color('white');
-    &:not(:last-of-type).has-divider {
-      border-bottom: 1px solid $divider-color;
-    }
+  &.selected ::ng-deep * {
+    font-weight: font-weight('bold');
+  }
+  &:focus {
```

Comment:
There is something wrong with the focus outline:

![image](https://user-images.githubusercontent.com/46445487/64951769-f1758300-d87e-11e9-95ff-543d290ffcbe.png)

And is it on on purpose outline is always shown and not just when the item is selectable?

Reply:
Don't think this is related to the PR...